scripts/utils.ts

```diff
@@ -46,13 +63,59 @@ export async function compileContracts(): Promise<CompilerOutput> {
 	return compilerOutput
 }
 
+function getScriptErrorMessage(errorCode: ScriptErrorCode, errorParameters?: string[]): string {
```

Comment:
nit: I think this should live in the workflow and not in typescript.

Reply:
Updated based on our discussion 👍🏾